core/sr-arithmetic/src/per_things.rs

```diff
@@ -0,0 +1,535 @@
+
+#[cfg(feature = "std")]
+use serde::{Serialize, Deserialize};
+
+use rstd::{ops, prelude::*, convert::TryInto};
+use codec::{Encode, Decode};
+use crate::traits::{SaturatedConversion, UniqueSaturatedInto, Saturating};
+
+/// Copied from `sr-primitives` and documented there.
+#[cfg(feature = "std")]
+#[cfg(test)]
+macro_rules! assert_eq_error_rate {
```

Comment:
this can probably be place somewhere nicer, no? 

We generally might need this for testing in this crate when there is a rounding error. So I would put it in `lib.rs`. 

And I don't know why we just thought of this 😁: why not import this into sr-primitives from here and just not define it there?

Reply:
also, I think cfg(test) is enough. `std` is out feature that explicitly says not wasm, i.e. native code. while `test` is also not-wasm, it is more expressive to say just `test`. This macro is and will never be used in any real code, just for test, regardless of wasm/std.

Reply:
> And I don't know why we just thought of this 😁: why not import this into sr-primitives from here and just not define it there?

Sorry, I'm not entirely sure what you're suggesting. Previously, I was defining `assert_eq_error_rate` in `sr-arithmetic` then re-exporting it in `sr-primitives`. Should we go back to doing that?

> also, I think cfg(test) is enough. std is out feature that explicitly says not wasm, i.e. native code. while test is also not-wasm, it is more expressive to say just test. This macro is and will never be used in any real code, just for test, regardless of wasm/std.
 
Done :^)

Reply:
> Sorry, I'm not entirely sure what you're suggesting. Previously, I was defining assert_eq_error_rate in sr-arithmetic then re-exporting it in sr-primitives. Should we go back to doing that?

Correct, I thought it is defined twice now. It is okay.